code/modules/clothing/suits/greatcoat.dm

```diff
@@ -80,7 +80,7 @@
 
 /obj/item/clothing/suit/greatcoat/nt_wintercoat //Sprite recolour from a Civ13 open github skyrim hidden piece with a few change ups to match our pallet -Dongels
 	name = "absolutist wintercoat"
-	desc = "A comfortably warm, and thick decorated wintercoat for the Absolutist faith and its supporters. Keeping the faithfull warm in the Amethyn heat since 2652."
+	desc = "A comfortably warm and thick decorated wintercoat for the Absolutist faith and its supporters. Keeping the faithful warm in the Amethyn heat since 2652."
```

Comment:
```suggestion
	desc = "A comfortably warm and thick decorated wintercoat for the absolutists and their supporters. Keeping the faithful warm in the Amethyn heat since 2652."
```